tests/test_news.py

```diff
@@ -1,37 +1,63 @@
 # -*- coding: utf-8 -*-
 """Testing that news entries are well formed."""
 import os
+import pytest
+import re
```

Comment:
PEP8 standard library imports should be first, then 3rd party packages, and then package or local imports.